toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -989,6 +989,7 @@ func ConfigureDiskBootloader(bootType string, encryptionEnable bool, readOnlyVer
 	partitionSettings []configuration.PartitionSetting, kernelCommandLine configuration.KernelCommandLine,
 	installChroot *safechroot.Chroot, diskDevPath string, installMap map[string]string,
 	encryptedRoot diskutils.EncryptedRootDevice, readOnlyRoot diskutils.VerityDevice, enableGrubMkconfig bool,
+	includeLegacyGrubCfg bool,
```

Comment:
Can't that be derived from enableGrubMkconfig? I assume we'd have either or?

Reply:
While that might be true, having it as a separate field makes it clear that this is functionality being kept around for the sake of the image customizer. So, I think it is better to keep it as is.